apps/src/pythonlab/PythonlabView.tsx

```diff
@@ -112,6 +111,30 @@ const PythonlabView: React.FunctionComponent<
   const isStartMode = getAppOptionsEditBlocks() === START_SOURCES;
 
   const currentLevel = useAppSelector(state => getCurrentLevel(state));
+  const dispatch = useAppDispatch();
+
+  const levelStartSources = useMemo(() => {
+    // For standalone project levels, we use the standalone start sources map to determine
```

Comment:
Nit: "For 'new' standalone project levels..." if you end up mapping below for projects that don't have sources defined yet.

Reply:
done!